blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
 * [Implementation steps](#implementation_steps "Goes to the Implementation steps section")
 * [Additional resources](#additional_resources "Goes to the Additional resources section")
 
 ## Solution components
 
-[comment]: # List components in logical order (for example, alphabetical, order of use in the solution, Genesys vs. third-party components, large components followed by smaller ones) Copy description from library, if description exists. Otherwise, follow format and brevity of descriptions in library. Format and examples follow. &&: we need to put library in public space.
-
-* **Genesys Cloud Bot Connector** - A set of RESTful APIs that allows you to connect to any bot platform to send and receive utterances. Available in beta and for digital messaging channels only. Please contact your Genesys Account or Partner Manager to enable this feature
-* **Microsoft Azure account** - To be able to create the Power VA bot
-* **AWS account** - To host the Lambda function that is acting as the Bot Interpreter application that is written in Python.
-* **REST client** - Something to submit a HTTP PUT with JSON payload to the Bot Connector API to create the botList. Postman, YARC or any application will work.
-* **A website to deploy Web Messaging** - To be able to test the final result.
+* **Genesys Bot Connector** - The Genesys Bot Connector configuration allows your third-party bots to interact conversationally with customers. A set of RESTful APIs that allows you to connect to any bot platform to send and receive utterances.
+*  **Architect flow** - A flow in Architect, a drag and drop web-based design tool, dictates how Genesys Cloud handles inbound or outbound interactions.
+* **Microsoft Azure account** - A cloud computing platform that provides a variety of cloud services for building, testing, deploying, and managing applications through Microsoft-managed data centers. Microsoft Azure hosts the Power VA bot.
+* **AWS** - Amazon Web Services, a cloud computing platform that provides a variety of cloud services such as computing power, database storage, and content delivery. AWS hosts Genesys Cloud.
+* **AWS Lambda** - A serverless computing service for running code without creating or maintaining the underlying infrastructure. In this solution, the Lambda function acts as the bot interpreter application that is written in Python.
+* **Amazon API Gateway** - An AWS service for using APIs in a secure and scalable environment. In this solution, the API Gateway exposes a REST endpoint that is protected by an API key. Requests that come to the API Gateway are forwarded to an AWS Lambda.
+* **REST API client** - A method or tool used to invoke REST API services that are exposed for communication. In this solution, you send an HTTP PUT request with JSON payload to the Bot Connector API that creates the bot list. For example, Postman, YARC or any other tool to send REST API requests and receive responses.
```

Comment:
How about: "A method or tool, such as Postman or YARC, that invokes exposed REST API services. In this solution, you send an HTTP PUT request with a JSON payload to the Bot Connector API to create the bot list. "

Reply:
If you agree, please add that description to the standards.